googleSpreadsheet.js

```diff
@@ -1,11 +1,12 @@
 const google = require('googleapis');
 const keys = require('./config/keys');
 
-function rowToObject(val, lab) {
+function rowToObject(val, lab, index) {
   const o = {};
   for (let i = 0; i < lab.length; i += 1) {
     o[lab[i]] = val[i];
   }
+  o.cellRef = 'https://docs.google.com/spreadsheets/d/1QHKa3vUpht7zRl_LEzl3BlUbolz3ZiL8yKHzdBL42dY/edit#gid=0&range=A' + index + ':T' + index;
   return o;
```

Comment:
potential security issue in leaking spreadsheet link to front end

hard coded value is pretty bad.

Reply:
agree, why do we need a cellRef here ?

Reply:
Not a necessary addition but, I thought having a link to any troublesome item's place in the spreadsheet would be useful for whoever has to edit it.

Reply:
that is a fair concern, and actually you can argue that google spreadsheet retrains access rights to the priviledged users. 

on further development we may need a better strategy for that spreadsheet bug fixing.

All that said I still approve that pull request